docs/index.md

```diff
@@ -0,0 +1,129 @@
+
+## Table of Contents
```

Comment:
Question: Many of the links in this TOC don't actually exist anymore. With the README getting too long, will the docs here get split up and cleaned up for accuracy as well? If you're considering fixing those up out of scope for this PR, is there a follow up todo for at minimum cleaning up the links, but preferably updating the docs entirely. I don't want to put in a PR killing the deadlinks and doing some cleanup/clarity just to hit merge conflicts. ;)

Suggestion for follow-up: Consider adding markdown-link-check to your build actions, to make sure that they stay up to date in the future?